### DISCUSSION TITLE
Beta testing serverless version

### DISCUSSION THREAD

**Discussion by qunash** [qunash/chatgpt-advanced#99] [Announcements]
Good news everyone!

👉 I've finished working on the serverless beta version of the extension (v3.0.0),
which enables web requests to be made locally in the browser without the need for a backend server.

This means that:
1. Your prompts are not sent to a 3rd party server.
2. Requests are processed faster.
3. I don't have to maintain a backend server.
It's a win all around! 😁

👉 As an exciting side-effect the extension now supports [duckduckgo bangs](https://duckduckgo.com/bangs)! 
👉 In the input field type `/` to open WebChatGPT commands menu.

ℹ️If you'd like to try this beta version, please follow the [instructions for manual installation](https://github.com/qunash/chatgpt-advanced/tree/serverless#manual-installation)

Don't forget to disable the extension you downloaded from the extension store when testing the beta version.

I would love to hear your feedback!

**Comment by Ignitius254** [comment]
How does chatgpt works
